File: Programming Basics with Python/04 PB Python for loop/More exercises/01_Back_To_The_Past.py
money = float(input())
year_to_live = int(input())
costs = 0
years = 17
for year in range(1800, year_to_live + 1):
    if year % 2 == 0:
        costs += 12000
        years += 1
    else:
        years += 1
        costs += 12000 + 50 * years
result = abs(money - costs)
if money >= costs:
    print(f"Yes! He will live a carefree life and will have {result:.2f} dollars left.")
else:
    print(f"He will need {result:.2f} dollars to survive.")

File: JS Basics/04 For-Loop - More Exercises/01. Back To The Past.js
function past(input1, input2) {
    let money = Number(input1);
    let yearToLive = Number(input2);
    let age = 18;
    let costs = 0;
    for (let i = 1800; i <= yearToLive; i++) {
        if (i % 2 == 0) {
            costs += 12000;
        }
        else {
            costs += 12000 + 50 * age;
        }
        age += 1;
    }
    let difference = Math.abs(money - costs);
    if (money >= costs) {
        console.log(`Yes! He will live a carefree life and will have ${difference.toFixed(2)} dollars left.`);
    }
    else {
        console.log(`He will need ${difference.toFixed(2)} dollars to survive.`);
    }
}
past("50000", "1802")